csi/controller_server.go

```diff
@@ -69,6 +69,8 @@ func NewControllerServer(apiClient *longhornclient.RancherClient, nodeID string)
 		accessModes: getVolumeCapabilityAccessModes(
 			[]csi.VolumeCapability_AccessMode_Mode{
 				csi.VolumeCapability_AccessMode_SINGLE_NODE_WRITER,
+				csi.VolumeCapability_AccessMode_SINGLE_NODE_SINGLE_WRITER,
+				csi.VolumeCapability_AccessMode_SINGLE_NODE_MULTI_WRITER,
```

Comment:
Do we want to support `VolumeCapability_AccessMode_SINGLE_NODE_MULTI_WRITER`?? 

> 	// Can be published as read/write at multiple workloads on a
	// single node simultaneously. SHOULD be used instead of
	// SINGLE_NODE_WRITER for COs using the experimental
	// SINGLE_NODE_MULTI_WRITER capability.
	VolumeCapability_AccessMode_SINGLE_NODE_MULTI_WRITER VolumeCapability_AccessMode_Mode = 7

https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/2485-read-write-once-pod-pv-access-mode/README.md#csi-specification-changes-volume-capabilities

`VolumeCapability_AccessMode_SINGLE_NODE_WRITER` might be sufficient in this case. Use `VolumeCapability_AccessMode_SINGLE_NODE_SINGLE_WRITER` until we support `VolumeCapability_AccessMode_SINGLE_NODE_MULTI_WRITER`.

cc @derekbit @james-munson

Reply:
Technically, we already support it at https://github.com/longhorn/longhorn-manager/blob/1e95ce6aab3aabc4f8acc4c0e3dbde1f95023c3c/csi/util.go#L454-L469, but we haven't really cared since then.

Reply:
Technically, we already support it at https://github.com/longhorn/longhorn-manager/blob/235b1ae23f6ad4d26dc56063c95358145e937c09/csi/util.go#L439-L454, but we haven't really cared until now.

Reply:
Oh nvm, this isn't the same the same capability